.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/java-8
+{
+	"name": "Java 8",
+    "image": "mcr.microsoft.com/devcontainers/java:1-8-bullseye",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				"java.import.gradle.java.home": "/usr/local/sdkman/candidates/java/current",
+				"java.configuration.runtimes": [{
+					"default": true,
+					"name": "JavaSE-1.8",
+					"path": "/usr/local/sdkman/candidates/java/current"
+				}]
+			},
+
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"vscjava.vscode-java-pack"
+			]
+		}
+	},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "java -version",
+
+	// Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+	"remoteUser": "vscode",
+	"features": {
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {},
+        "ghcr.io/devcontainers/features/java:1": {
+            "version": "none",
+            "installMaven": "false",
+            "installGradle": "false"
+        },
+        "ghcr.io/devcontainers/features/docker-from-docker:1": {},
```

Comment:
```suggestion
        "ghcr.io/devcontainers/features/docker-from-docker:2": {},
```

Reply:
AFAIK, there is no version 2. I just noticed it has been move to [docker-outside-of-docker](https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/README.md)

Reply:
There is: https://github.com/devcontainers/features/blob/f7314671e1654cd47263a09b134ddd0286f9eccc/src/docker-in-docker/devcontainer-feature.json#L3.

Reply:
Sorry, in the first commit I mixed it up with `docker-in-docker:2`.